widget/ui/src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,58 @@
+import type { PropTypes } from './Tabs.types';
+
+import React from 'react';
+
+import { Divider } from '../Divider';
+import { Tooltip } from '../Tooltip';
+
+import { BackdropTab, Tab, Tabs } from './Tabs.styles';
+
+export function TabsComponent(props: PropTypes) {
+  const {
+    Items,
+    onChange,
+    container = document.body,
+    value,
+    currentTabIndex,
+    tabWidth,
+    type,
+    border,
+  } = props;
+  return (
+    <Tabs type={type} border={border}>
+      {Items.map((item, index) => (
+        <Tooltip
+          key={item.id}
+          style={{ width: '100%' }}
+          container={container}
+          side="bottom"
+          sideOffset={2}
+          content={item.tooltip}
+          open={!item.tooltip ? false : undefined}>
```

Comment:
I don't get this line of code: "open={!item.tooltip ? false : undefined}>"
how can it be open?

Reply:
- If there is tooltip content available (item.tooltip is truthy), open will be set to undefined, letting RadixTooltip decide when to display it.
 - If there's no tooltip content (item.tooltip is falsy), open will be set to false, explicitly preventing the tooltip from opening.